ui/page/send/page.go

```diff
@@ -44,215 +26,41 @@ type Page struct {
 	ctx       context.Context // page context
 	ctxCancel context.CancelFunc
 
-	pageContainer *widget.List
-
-	sourceAccountSelector *components.WalletAndAccountSelector
-	sendDestination       *destination
-	amount                *sendAmount
-
-	infoButton    cryptomaterial.IconButton
-	retryExchange cryptomaterial.Button
-	nextButton    cryptomaterial.Button
-
-	shadowBox *cryptomaterial.Shadow
-	backdrop  *widget.Clickable
-
-	isFetchingExchangeRate bool
-
-	exchangeRate        float64
-	usdExchangeSet      bool
-	exchangeRateMessage string
-	confirmTxModal      *sendConfirmModal
-	currencyExchange    string
-
-	txLabelInputEditor cryptomaterial.Editor
-
-	*authoredTxData
-	selectedWallet  *load.WalletMapping
-	feeRateSelector *components.FeeRateSelector
-
-	toCoinSelection *cryptomaterial.Clickable
-
-	selectedUTXOs selectedUTXOsInfo
```

Comment:
I think these are moved to the sharedProperties struct. Why can't they be left untouched? Fewer changes make for easier review and the less likelyhood of a retrogression.

Reply:
Modals do not implement OnNavigatedTo, and Page.Layout().
Modal use OnResume and the likes.

We cannot render *Page Layout on Modal.
But since the pages share same page content, i moved the shared codes and call them where they are needed on the Send Page and Send Modal.

Reply:
The codes did not change.. i just moved them to a shared file.
I could move the share codes back to page is thats what you want and send page would go into a new file.. its same thing.